org/ystia/alien/docker/types.yaml

```
@@ -0,0 +1,58 @@
+tosca_definitions_version: alien_dsl_2_0_0
+#
+# Copyright 2020 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+metadata:
+  template_name: org.ystia.alien.docker
+  template_version: 1.0.0-SNAPSHOT
+  template_author: yorc
+
+imports:
+  - tosca-normative-types:1.0.0-ALIEN20
+  - docker-types:2.2.0-SM10
```

Is it correct to use a specific version here ?
Shouldn't we use the snapshot version instead ?
```suggestion
  - docker-types:2.2.0-SNAPSHOT
```

---

SM10 allows for testing on Alien 2.2.0-SM10. 
We could change this for SNAPSHOT, however its explained in the README.md file that user should change the docker-types version if his Alien has another version then 2.2.0-SM10, before creating the CSAR and uploading it in Alien